utils/k8s/openebscrd.go

```diff
@@ -40,3 +40,33 @@ func (k8s K8S) DeleteStoragePoolClaim(spcName string) error {
 	spcClient := k8s.OpenebsClientSet.Openebs().StoragePoolClaims()
 	return spcClient.Delete(spcName, &meta_v1.DeleteOptions{})
 }
+
+// ListDisks  list All disks
+func (k8s K8S) ListDisks() (*openebs_v1.DiskList, error) {
+	diskClient := k8s.OpenebsClientSet.OpenebsV1alpha1()
+	diskList, err := diskClient.Disks().List(meta_v1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return diskList, nil
+}
+
+// GetDisk ... get disk with diskname
```

Comment:
@MuthukumarHelios 
Can we remove the `...` thing.
It should be following:
`GetDisk  gets disk with diskname.`